Firmware/RoboPad/RoboPad.ino

```diff
@@ -423,22 +432,64 @@ void setup() {
   // Set up the websocket server:
   webSocketServer.begin();
   webSocketServer.onEvent(webSocketEvent);
+  Serial.println("WebSocket server started.");
 
   // Configure IO
   configureIO();
+
+  attachInterrupt(PPM_PIN, interruptPPM, CHANGE);
+  Serial.println("PPM interrupt attatched");
```

Comment:
These have been missed from the PPM ifdef.

Reply:
Fixed this in latest commit